scripts/parse-contributors.py

```diff
@@ -0,0 +1,62 @@
+import os
+
+from pyosmeta.contributors import ProcessContributors
+
+# API_TOKEN = get_api_token()
+API_TOKEN = os.environ.get("GITHUB_TOKEN")
+
+json_files = [
+    "https://raw.githubusercontent.com/pyOpenSci/python-package-guide/main/.all-contributorsrc",
+    "https://raw.githubusercontent.com/pyOpenSci/software-peer-review/main/.all-contributorsrc",
+    "https://raw.githubusercontent.com/pyOpenSci/pyopensci.github.io/main/.all-contributorsrc",
+    "https://raw.githubusercontent.com/pyOpenSci/software-review/main/.all-contributorsrc",
+    "https://raw.githubusercontent.com/pyOpenSci/update-web-metadata/main/.all-contributorsrc",
+    # "https://raw.githubusercontent.com/pyOpenSci/examplepy/main/.all-contributorsrc",
+]
+
+# Get contribs from pyopensci.github.io repo (this is what is published online)
+web_yaml_path = "https://raw.githubusercontent.com/pyOpenSci/pyopensci.github.io/main/_data/contributors.yml"
+
+# Instantiate contrib object
+processContribs = ProcessContributors(json_files, web_yaml_path, API_TOKEN)
+
+# Returns a list of dict objects with gh usernames (lowercase) as keys
+web_yml_dict = processContribs.load_website_yml()
+bot_all_contribs_dict = processContribs.combine_json_data()
+
+# Create a single dict containing both website and all-contrib bot users
+
+all_contribs_dict = processContribs.combine_users(
+    bot_all_contribs_dict, web_yml_dict
+)
+
+# Force gh username keys to be lowercase
+for key in all_contribs_dict:
+    all_contribs_dict[key]["github_username"] = all_contribs_dict[key][
+        "github_username"
+    ].lower()
+    print(all_contribs_dict[key]["github_username"])
```

Comment:
Why not just

```suggestion
for key, user_dict in all_contribs_dict.items():
    user_dict["github_username"] = user_dict["github_username"].lower()
    print(user_dict["github_username"])
```

Reply:
oh much cleaner. i will be able to remove a lot of print statements once this is working. i had them in there for troubleshooting early on. there are too many in the package as well :)  but i just want this pr to work first.